dotcom-rendering/src/web/components/CardHeadline.tsx

```diff
@@ -84,14 +84,20 @@ const labTextStyles = (size: SmallHeadlineSize) => {
 	}
 };
 
-const underlinedStyles = (size: SmallHeadlineSize) => {
+const underlinedStyles = (size: SmallHeadlineSize, colour: string) => {
+	// extract hex colours as numbers
+	const r = parseInt(colour.slice(1, 3), 16);
+	const g = parseInt(colour.slice(3, 5), 16);
+	const b = parseInt(colour.slice(5, 7), 16);
+	const transparentColour = `rgba(${r}, ${g},${b}, 0.5)`;
+
 	function generateUnderlinedCss(baseSize: number) {
 		return css`
 			background-image: linear-gradient(
 				to bottom,
 				transparent,
 				transparent ${baseSize - 1}px,
-				rgba(199, 0, 0, 0.5)
+				${transparentColour}
 			);
```

Comment:
@oliverlloyd since you’ve review I’ve proposed this. Can split in its own PR if it needs more work.

Reply:
Added a new branch. Let’s not confuse things.